src/sass/modules/_m-alert.scss

```diff
@@ -17,7 +17,7 @@
 }
 
 .va-alert-body {
-  padding-left: 2.5rem;
+  padding-left: 3.5rem;
```

Comment:
This looks like it's for `padding-left`, not `padding-right` as the issue indicates.